code/game/gamemodes/game_mode.dm

```diff
@@ -34,7 +34,6 @@
 	var/newscaster_announcements = null
 	var/ert_disabled = FALSE
 	var/uplink_welcome = "Syndicate Uplink Console:"
-	var/uplink_uses = 100
```

Comment:
You still got unrelated changes

Reply:
The uplink uses being declared in gamemode instead of uplink meant that uplinks placed down before roundstart would cause a runtime. The uplink file was the only thing I could find that references it, so I moved it over. This was mentioned in the 'what does this pr do' section.